certified-connectors/Tikit/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2652 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "Tikit",
+    "description": "Extend conversational ticketing built for Microsoft Teams with activities to create tickets in response to events or update tickets with more information as context changes.",
+    "contact": {
+      "name": "Tikit Support",
+      "url": "https://help.tikit.ai/",
+      "email": "help@tikit.ai"
+    }
+  },
+  "host": "app.tikit.ai",
+  "basePath": "/api/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "multipart/form-data"
+  ],
+  "produces": [],
+  "paths": {
+    "/Ticket": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {}

```

Comment:
@TikitTeam Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger. **This applies to all the empty schemas in the swagger.**

Reply:
updated